src/main/java/de/tum/in/www1/artemis/service/ResultService.java

```diff
@@ -210,7 +210,9 @@ public void createNewManualResult(Result result, boolean isProgrammingExerciseWi
                 Hibernate.initialize(savedResult.getParticipation().getExercise());
             }
 
-            ltiService.onNewBuildResult((ProgrammingExerciseStudentParticipation) savedResult.getParticipation());
+            if (savedResult.getParticipation() instanceof ProgrammingExerciseStudentParticipation) {
```

Comment:
I think here we also have to watch out that template or solution participations don't enter this area.